src/data/bitvec/core.lean

```diff
@@ -0,0 +1,219 @@
+/-
+Copyright (c) 2015 Joe Hendrix. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joe Hendrix, Sebastian Ullrich
+
+Basic operations on bitvectors.
+
+This is a work-in-progress, and contains additions to other theories.
```

Comment:
```suggestion
```
Perhaps we can drop this from a file that has been frozen in core for so long! :-)

Reply:
Well, it was not well tested and when I needed a bitvec library this one didn't really work for me so I had to write my own. So I wouldn't say that time has really tested the library so much as broken it.

Reply:
I wondered whether it might be worth acknowledging its status as a file moved wholesale from the core library, but it seems like we haven't been recording this information, other than by inference from the date/author data.

Reply:
```suggestion
This is a work-in-progress, and contains additions to other theories.

This file was moved to mathlib from core Lean in the switch to Lean 3.20.0c. It is not fully in compliance with mathlib style standards.
```